gems/redis/4.2/_test/get_set.rb

```diff
@@ -5,3 +5,7 @@
 
 redis.get("mykey")
 # => "hello world"
+
+Redis.current.set(:mykey, 'hello world')
```

Comment:
https://github.com/redis/redis-rb/blob/d6626e6ef0c49374a888074652c4c9b2237dc8eb/lib/redis.rb#L39-L46

`Redis.current` is deprecated. So could you use `Redis.new` instead in this test code?


And the added type for `Redis.current` can be removed also. Personally, I think we do not need to add a type definition for a deprecated method actively. Because a deprecated method should not be used, so we can omit to add it to avoid misusing the deprecated method.

---


By the way, RBS does not have any rule for deprecated methods yet. I'd like to introduce something to mark a type deprecated, such as an annotation. But there is no implementation for now.

Reply:
Thank you for your review.

There are no rules regarding deprecated methods in RBS, and you basically have a policy of not adding types.
I understand.

I'll fix the test and remove the `Redis.current` type definition.